docs/Project-Configuration/Project-Commands.md

```diff
@@ -379,10 +381,18 @@ The parameters for each module are:
 -   `prefix`: the subdirectory to clone the repository in. It will be
     the repository name as a top-level directory in `dir` if omitted
 -   `ref`: must be a commit hash, takes precedence over the `branch`
-    parameter if both specified
+    parameter if both specified (for commits)
 -   `branch`: must be the name of branch, commit hashes _are not
     accepted_.
 
+
+More specifically, module hash priority is as follows:
+* For commit queue patches, we always use the module branch name, to ensure accurate testing.
```

Comment:
I'm not that familiar with how modules interact with the commit queue. Does this comment + the above docs about `branch` mean that if your project uses teh commit queue and you have modules, you must explicitly set `branch` in the YAML, or does it get auto-populated (by set-module or something)?

Reply:
It looks like we expect branch: https://github.com/evergreen-ci/evergreen/blob/96b3f970f8e436d639cc8ee19456d225dda3384d/validator/project_validator.go#L984

I'm not sure if the new commit queue is set up to handle modules right. I can file this away as a question for Brian when he's back.

Reply:
Sounds good, thanks!